cli/cmd/env/configure.go

```diff
@@ -51,7 +51,7 @@ func ConfigureCmd(ch *cmdutil.Helper) *cobra.Command {
 				var err error
 				projectName, err = ch.InferProjectName(ctx, ch.Org, projectPath)
 				if err != nil {
-					return err
+					return fmt.Errorf("unable to infer project name, use `--project` to specify the project name")
```

Comment:
Are you sure it's safe to ignore the `err`? What if they are in the right Git repo, but there's a network error or permission error?

Reply:
Yes. IMO inference of project name is best effort and if it fails(mostly will succeed with `.rillcloud` being checked in now) best is to ask users to pass project name. 
For permissions the source of truth is permissions that are stored in rill so a user with no access to github repo but access to rill project should be able to make changes ?

Reply:
> mostly will succeed with .rillcloud being checked in now

Actually I think this is the problem – it will fail quite often now because someone else clones the repo, and for example tries to run Rill commands, but maybe hasn't run `rill login` yet. Then they will get an auth required error.

I think it might be better to still return the original error here along with the hint, like:
```go
fmt.Errorf("unable to infer project name (use `--project` to explicitly specify the name): %w", err)
```

Reply:
Sure. Original error with `--project` message sounds good.